modules/user/user.service.js

```diff
@@ -105,6 +105,45 @@ class UserService {
     );
   }
 
+  async loginAdmin({ email, password }) {
+    const { errors } = await validateLoginInput.validateAsync({
+      email,
+      password,
+    });
+
+    if (errors) {
+      throw new UserInputError('Errors', {
+        errors,
+      });
+    }
+
+    const user = await this.getUserByFieldOrThrow('email', email);
+
+    const match = await bcrypt.compare(
+      password,
+      user.credentials.find(cred => cred.source === 'horondi').tokenPass,
```

Comment:
Please, move the string `horondi` to some enum, create object named `SOURCES` and store value `horondi` there to ensure that no typo can happen

Reply:
Moved!